packages/terra-popup/tests/wdio/popup-spec.js

```diff
@@ -1,499 +1,360 @@
-    beforeEach(() => {
+    it('adjusts the arrow for vertical-left attachment when arrow would be offscreen', () => {
       browser.url('/#/raw/tests/terra-popup/popup/arrow-vertical-left-adjustment-popup');
       browser.waitForVisible('.test-content');
+      Terra.validates.screenshot('adjusted vertical-left', { selector });
     });
 
-    Terra.it.matchesScreenshot({ selector: '#test-popup-area' });
-  });
-
-  describe('Adjusts the arrow for vertical-right attachment when arrow would be offscreen', () => {
-    before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-    beforeEach(() => {
+    it('adjusts the arrow for vertical-right attachment when arrow would be offscreen', () => {
       browser.url('/#/raw/tests/terra-popup/popup/arrow-vertical-right-adjustment-popup');
       browser.waitForVisible('.test-content');
+      Terra.validates.screenshot('adjusted vertical-right', { selector });
     });
-
-    Terra.it.matchesScreenshot({ selector: '#test-popup-area' });
   });
 
-  describe('Applies content offset when target has vertial-right when content vertial-left attachment', () => {
-    before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-popup/popup/different-attachments-popup');
+  describe('Content offset', () => {
+    before(() => browser.url('/#/raw/tests/terra-popup/popup/different-attachments-popup'));
+
+    it('applies content offset when target has vertical-right when content vertical-left attachment', () => {
       browser.click('#attach-Left');
       browser.waitForVisible('.test-content');
+      Terra.validates.screenshot('target:vertical-right & content:vertical-left', { selector });
+      browser.keys('Escape');
     });
 
-    Terra.it.matchesScreenshot({ selector: '#test-popup-area' });
-  });
-
-  describe('Applies content offset when target has vertial-left when content vertial-right attachment', () => {
-    before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-popup/popup/different-attachments-popup');
+    it('applies content offset when target has vertical-left when content vertical-right attachment', () => {
       browser.click('#attach-Right');
       browser.waitForVisible('.test-content');
+      Terra.validates.screenshot('target:vertical-left & content:vertical-right', { selector });
+      browser.keys('Escape');
     });
-
-    Terra.it.matchesScreenshot({ selector: '#test-popup-area' });
   });
 
-  describe('Applies content offset when target is smaller than the arrow for vertial-left attachment', () => {
-    before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-    beforeEach(() => {
+  describe('Popup with [Target Smaller Than Arrow]', () => {
+    before(() => browser.url('/#/raw/tests/terra-popup/popup/different-attachments-popup'));
```

Comment:
is this url necessary? i see we immediately hit a different url in the next it block

Reply:
no. Looks like a copy & paste mistake. Good catch

Reply:
removed: https://github.com/cerner/terra-framework/pull/734/commits/0afcefd17e8a4c5e3f0b94be3d8fd1ad28cbd347#diff-75a22fd845592219495fcc77ea6104c3L190